src/main/scala/cromwell/binding/values/WdlArray.scala

```diff
@@ -34,4 +37,10 @@ case class WdlArray(wdlType: WdlArrayType, value: Seq[WdlValue]) extends WdlValu
   override def collectAsSeq[T <: WdlValue](filterFn: PartialFunction[WdlValue, T]): Seq[T] = {
     value flatMap { _.collectAsSeq(filterFn) }
   }
-}
\ No newline at end of file
+
+  override def getHash(implicit hasher: FileHasher) = {
```

Comment:
it's part of the signature in `WdlValue`, it is used in most of the implementers